contents/docs/advanced/_snippets/region-warning.mdx

```diff
@@ -0,0 +1 @@
+> NOTE: If you are using the EU cloud then use `eu` instead of `us` in all domains (e.g. `us.i.posthog.com` -> `eu.i.posthog.com`)
```

Comment:
```suggestion
> **Note:** If you are using the EU cloud then use `eu` instead of `us` in all domains (e.g. `us.i.posthog.com` -> `eu.i.posthog.com`)
```